src/structures/Guild.js

```diff
@@ -182,6 +182,12 @@ class Guild extends Base {
      */
     this.joinedTimestamp = data.joined_at ? new Date(data.joined_at).getTime() : this.joinedTimestamp;
 
+    /**
+     * The value set for a guild's default message notifications
+     * @type {DefaultMessageNotifications}
+     */
+    this.defaultMessageNotifications = DefaultMessageNotifications[data.default_message_notifications];
```

Comment:
If discord decides to add another type this should fallback to the original number received.
Adding something like `|| data.default_message_notifications` should do it.

Reply:
Done! And updated type to be `DefaultMessageNotifications|number` for the case it fallbacks.